crates/runtime/src/backtrace/aarch64.rs

```diff
@@ -0,0 +1,39 @@
+// The aarch64 calling conventions save the return PC one word above the FP and
```

Comment:
One small detail - `word` is a loaded term in the Arm architecture, and in this context certainly wrong, since it means 32 bits (the AAPCS64 also provides the same definition in section 5.1); up to you whether you use the architecture-neutral `64 bits` or the correct `double-word`.

Reply:
FWIW these terms are loaded in x86-land as well: "word" means 16 bits (still to this day, e.g. with a `.word` directive in GNU as assembler), "double word" or "dword" means 32 bits, and "quad word" or "qword" means 64 bits. These names appear at least in assembly but also e.g. in Windows APIs, and differ between architectures as we see here, and so I try to avoid them altogether and just say "64 bits" / "u64" / "8 bytes above FP" / ... as appropriate for context.

Reply:
Personally I am fine with using the loaded terms in bits of code that are obviously architecture-specific such as this file, but have no preference otherwise, other than using the correct term, of course. Note that we are providing a direct quote from the AAPCS64 that uses both kinds of terms.